src/main/java/com/github/shyiko/mysql/binlog/event/deserialization/AbstractRowsEventDataDeserializer.java

```diff
@@ -317,6 +491,34 @@ protected Serializable deserializeTimeV2(int meta, ByteArrayInputStream inputStr
         return timestamp != null ? convertLongTimestamptWithFSP(timestamp, fsp) : null;
     }
 
+    protected Serializable deserializeTimeV2(int meta, BinaryLogEventDataReader eventDataReader) {
```

Comment:
FYI there is another time deserializer to update: https://github.com/fivetran/mysql-binlog-connector-java-osheroff/commit/dc527cbc54ceb046736c89f860aa69c6ffb557a0